Source/PeleC.cpp

```diff
@@ -1758,6 +1758,18 @@ PeleC::errorEst(
           tilebox, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
             tag_error_bounds(i, j, k, tag_arr, vfrac_arr, 0.0, 1.0, tagval);
           });
+
+        const int local_i = mfi.LocalIndex();
+        const int Nebg = (!eb_in_domain) ? 0 : sv_eb_bndry_geom[local_i].size();
+        EBBndryGeom* ebg = sv_eb_bndry_geom[local_i].data();
+        const auto lo = amrex::lbound(tilebox);
+        const auto hi = amrex::ubound(tilebox);
+        amrex::ParallelFor(Nebg, [=] AMREX_GPU_DEVICE(int L) {
+          const auto& iv = ebg[L].iv;
+          if (is_inside(iv, lo, hi)) {
+            tag_arr(iv) = tagval;
+          }
+        });
```

Comment:
Why can't we just run through the list of ebbndrygeom objects and tag if tilebox.contains(ebg[L].iv)? Not sure  understand why we have to do all that other checking.

Reply:
It's what happens when you do this between meetings ;) I updated the code. Is that what you meant?

Reply:
Yeah, exactly!